README.md

```diff
@@ -188,22 +188,22 @@ public class MailClientHandlerFactory : IMailClientHandlerFactory
     public IMailClient CreateMailClient(HttpRequest request)
     {
         request.TryParseCustomHeaderValueIntoBoolean(
-            _smtpOptions.DontForwardMessageToProviderCustomHeaderName ?? string.Empty,
-            out bool dontForwardMessageToProvider
+            _smtpOptions.EmailDryRunHeaderName ?? string.Empty,
+            out bool emailDryRun
             );
 
-        return dontForwardMessageToProvider
-            ? (IMailClient)Activator.CreateInstance(typeof(NoopMailClient))
+        return emailDryRun
+            ? new NoopMailClient()
             : _smtpOptions.EmailClientType switch
             {
-                EmailClientType.None => (IMailClient)Activator.CreateInstance(typeof(NoopMailClient)),
-                _ => (IMailClient)Activator.CreateInstance(typeof(MailKitClient), _smtpOptions)
+                EmailClientType.None => new NoopMailClient(),
+                _ => new MailKitClient(_smtpOptions)
             };
     }
 }
 ```
 
-Extract `IMailClientHandlerFactory` from this class. Using DI add this interfcase as singleton.
+Extract `IMailClientFactory` from this class. Using DI add this interfcase as singleton.

```

Comment:
Typo: "interfcase" (presumably should be "interface").

Reply:
fixed and I hate my keyboard 🤣